UnitySetup/UnitySetup.psm1

```diff
@@ -492,6 +492,7 @@ function Install-UnitySetupInstance {
                 }
             }
         }
+        [Environment]::SetEnvironmentVariable("UNITY_"+$version, $destPath, "Unity Installer Path")
```

Comment:
I think we may want to use `[System.EnvironmentVariableTarget]::User` as the last argument.

Reply:
We also should only run this when the install is successful.

Reply:
Could you tell me how to check if the install is successfull?
Should I check for `$process` ?

Reply:
Yes. If you look at line 491 you should see the message where we print success. I would think we just move this line right before that.